api/src/main/java/org/apache/cloudstack/api/response/NetworkResponse.java

```diff
@@ -238,6 +238,10 @@
     @Param(description = "If the network has redundant routers enabled", since = "4.11.1")
     private Boolean redundantRouter;
 
+    @SerializedName(ApiConstants.ACL_NAME)
+    @Param(description = "ACL name associated with the VPC network")
```

Comment:
@ravening will it be better to add a since tag here?

Reply:
@shwstppr made the change